main.tf

```diff
@@ -95,12 +95,22 @@ resource "azurerm_linux_virtual_machine" "linux" {
     }
   }
 
+  boot_diagnostics {
+    storage_account_uri = var.diagnostics_storage_account_uri
+  }
+
   os_disk {
     caching                   = var.operating_system_disk_cache
     storage_account_type      = var.operating_system_disk_type
     write_accelerator_enabled = var.operating_system_disk_write_accelerator
   }
 
+  additional_capabilities {
```

Comment:
This block should probably be conditionalized

Reply:
changed to a dynamic block